spec/src/main/asciidoc/microprofile-lra-spec.adoc

```diff
@@ -335,14 +337,14 @@ security is specifically not defined in the specification and it is up to implem
 [[lra-context]]
 ==== The LRA Context
 
-When a method is invoked in the context of an LRA its identifier (of type java.net.URI) MUST be made
+When a method (that is annotated with any of the annotations defined in this specification)
```

Comment:
I would suggest rephrasing this to "a method that is invoked by the implementation of this specification" because we also include annotations (LRA, Leave) that are executed by users and thus we can't always guarantee that the LRA id is made available.

Reply:
We still need to make the context available even when clients invoke the methods. For example it is how we implement LRA element types such as REQUIRED, REQUIRES_NEW, etc. An example implementation technique could be to apply a JAX-RS filter to outgoing requests, this filter would add the context header making it available at the target service. The target service would have another incoming JAX-RS filter which would decide what to do with the context before the business method runs.